components/schemes/README.md

```diff
@@ -10,119 +10,195 @@ path: /catalog/theming/
 
-## Overview
+1. Components
+2. Subsystem Schemes
+3. The Container Scheme
+4. Theming Extensions
 
-Material Theming is a consistent way to apply a uniform design across your app when using Material Components for iOS. Material Theming on iOS consists of two primary patterns: schemes and themers.
+Components are expected to provide public APIs for a variety of parameters. An example of a component is [MDCButton](https://github.com/material-components/material-components-ios/tree/develop/components/Buttons).
 
-- **Schemes** represent your design as systemized symbols.
-- **Themers** are the glue that apply scheme symbols to components.
+Subsystem schemes represent a set of opinionated properties that are intended to be mapped to component parameters. There is a scheme for each Material Theming subsystem. For example, there is a scheme for the color, shape, and typography subsystems.
 
-For example, there is a scheme for both the Material Design typography system and the Material
-Design color system. Most components have a themer for at least one of these systems.
+The Container scheme represents a single configurable entity that is applicable to all themeable components. A container scheme consists of all of the subsystem schemes.
+
+Theming extensions are component extensions that, when invoked with a default container scheme, will theme a component according to the [Material Design guidelines](https://material.io/design). When provided with subsystem schemes via a container scheme, the extension will map the subsystem scheme's values to the component’s parameters.
 
 ### Sensible defaults, yet highly configurable
 
-By default, an instance of a scheme is initialized with the Material defaults. You can use these
-defaults as a baseline, but at a minimum we encourage you to tweak your color scheme's primary and
-secondary colors to match your brand colors.
+By default, components have reasonable defaults for all of their customizable properties, e.g. `backgroundColor` or `titleFont`. You can use these defaults as a baseline, but we encourage you to theme your components to match your brand style using theming extensions.
 
 ### Schemes
 
 <ul class="icon-list">
-  <li class="icon-list-item icon-list-item--link"><a href="Color/">Color scheme</a></li>
-  <li class="icon-list-item icon-list-item--link"><a href="Shape/">Shape scheme</a></li>
-  <li class="icon-list-item icon-list-item--link"><a href="Typography/">Typography scheme</a></li>
+<li class="icon-list-item icon-list-item--link"><a href="Container/">Container scheme</a></li>
+<li class="icon-list-item icon-list-item--link"><a href="Color/">Color scheme</a></li>
+<li class="icon-list-item icon-list-item--link"><a href="Shape/">Shape scheme</a></li>
+<li class="icon-list-item icon-list-item--link"><a href="Typography/">Typography scheme</a></li>
 </ul>
 
 ## Examples
 
-### Creating a scheme
+### Theme component
```

Comment:
Should this be "Theming a Component" or are you referring to the "Themes" component in `components/Themes`?

Reply:
Fixed